src/include/api.h

```diff
@@ -67,6 +67,12 @@
     (s)->name = __oldname;  \
     --(s)->api_call_counter
 
+#define API_NAME_PUSH(s, struct_name, func_name) \
```

Comment:
Can we have a comment here calling out that API_NAME_PUSH is a lighterweight version of API_SESSION_PUSH?

Reply:
done